lib/server.js

```diff
@@ -384,7 +384,8 @@ Server.prototype.start = function start(cb) {
     debug('emitting listening event');
     var addr = self._httpServer.address();
     self.emit('listening', addr);
-    self._meshApp.emit('started', self.port);
+    self._meshApp.set('url', util.format('http://127.0.0.1:%d', addr.port));
```

Comment:
I don't know what `url` is... but is this right? It is used only from localhost?